app/src/main/java/com/github/andrezimmermann/gluappsample/client/list/BusLineSelecionHandler.java

```diff
@@ -0,0 +1,10 @@
+package com.github.andrezimmermann.gluappsample.client.list;
+
+import com.github.andrezimmermann.gluappsample.client.mvp.event.ListItemSelectionHandler;
+import com.github.andrezimmermann.gluappsample.shared.data.BusLine;
+
+/**
+ * TODO: add javadoc
+ */
+public interface BusLineSelecionHandler extends ListItemSelectionHandler<BusLine> {
```

Comment:
Why have you created this interface? I think that passing the BusLine type to the ListItemSelectionHandler is enough to indicate that it is BusLine handler, isn't?